app/src/main/java/com/example/myfridgehome/dto/SetStoredFoods.kt

```diff
@@ -11,7 +12,7 @@ import java.io.StringWriter
 
 
 //this class collects the user input and converts it into a writable string format.
-abstract class SetStoredFoods(var name: String, var type: String, var quantity : Int, var measurement : String)
+abstract class SetStoredFoods(@SerializedName("name") var name: String, @SerializedName("type") var type: String, @SerializedName("quantity") var quantity : Int, @SerializedName("measurement") var measurement : String)
```

Comment:
@SerializedName is an annotation that indicates the annotated member should be serialized to JSON with the provided name value as its field name. It will give nice formatting.

Reply:
@SerializedName is only required if the JSON name is different than the variable/attribute name.  Since each of these @SerializedName annotations are simply repeating the attribute name, they are not adding value... just more typing.  I'd keep it simple, and stick with the original version, and only add @SerializedName where required to map unmatched names.